### PR TITLE
Remove JFrog downtime notice

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -4,12 +4,6 @@ In a nutshell, Jenkins is the leading open-source automation server.
 It provides over 1,800 [plugins](https://plugins.jenkins.io/) for many use cases so that humans can spend their time doing things machines cannot.
 Learn more: [Jenkins Website](https://www.jenkins.io/), [Documentation](https://www.jenkins.io/doc/), [Plugin Index](https://plugins.jenkins.io/)
 
-## Scheduled downtime for Jenkins Artifactory and ci.jenkins.io
-
-The Jenkins artifact repository, [repo.jenkins-ci.org](https://repo.jenkins-ci.org/), will be upgraded beginning Sunday, December 18, 2022 at 9:00 AM UTC.
-Downtime is expected to be 6 hours or less.
-More information is available on [status.jenkins.io](https://status.jenkins.io/issues/2022-12-09-artifactory-maintenance/) and in the [announcement blog post](https://www.jenkins.io/blog/2022/12/09/jfrog-maintenance/).
-
 ##
```
Comment (lemeurherve) [inline profile/README.md:8]:
```suggestion
```

**Review by lemeurherve — CHANGES_REQUESTED** [review]
LGTM, need to remove 2 more lines though

File: profile/README.md [hunk 2]
```diff
@@ -4,12 +4,6 @@ In a nutshell, Jenkins is the leading open-source automation server.
...
```
Comment (NotMyFault) [inline profile/README.md:7]:
```suggestion
```

File: profile/README.md [hunk 3]
```diff
@@ -4,13 +4,6 @@ In a nutshell, Jenkins is the leading open-source automation server.
 It provides over 1,800 [plugins](https://plugins.jenkins.io/) for many use cases so that humans can spend their time doing things machines cannot.
 Learn more: [Jenkins Website](https://www.jenkins.io/), [Documentation](https://www.jenkins.io/doc/), [Plugin Index](https://plugins.jenkins.io/)
 
-## Scheduled downtime for Jenkins Artifactory and ci.jenkins.io
-
-The Jenkins artifact repository, [repo.jenkins-ci.org](https://repo.jenkins-ci.org/), will be upgraded beginning Sunday, December 18, 2022 at 9:00 AM UTC.
-Downtime is expected to be 6 hours or less.
-More information is available on [status.jenkins.io](https://status.jenkins.io/issues/2022-12-09-artifactory-maintenance/) and in the [announcement blog post](https://www.jenkins.io/blog/2022/12/09/jfrog-maintenance/).
-
-##
```
Comment (lemeurherve) [inline profile/README.md:7]:
```suggestion
```
One more time ^^

**Review by lemeurherve — APPROVED** [review]

**Comment by NotMyFault** [comment]
cc @timja or someone else who could merge this PR 👀

**Pull request merged**